packages/commands/src/auth-headers.mjs

```diff
@@ -1,27 +1,37 @@
-import chalk from "chalk";
-import { program } from "commander";
+import chalk from 'chalk'
+import { program } from 'commander'
 
-import { initializeAudiusLibs } from "./utils.mjs";
+import { initializeAudiusSdk } from './utils.mjs'
 
 program
-  .command("auth-headers")
+  .command('auth-headers')
   .description(
-    "Output auth headers (for use with curl: `curl -H @<(audius-cmd auth-headers)`)"
+    'Output auth headers (for use with curl: `curl -H @<(audius-cmd auth-headers)`)'
   )
-  .option("-f, --from [from]", "The account for which to generate auth headers")
+  .option('-f, --from [from]', 'The account for which to generate auth headers')
   .action(async ({ from }) => {
-    const audiusLibs = await initializeAudiusLibs(from);
+    const audiusSdk = await initializeAudiusSdk({ handle: from })
 
     try {
-      const unixTimestamp = Math.round(new Date().getTime() / 1000);
-      const message = `Click sign to authenticate with identity service: ${unixTimestamp}`;
-      const signature = await audiusLibs.web3Manager.sign(message);
+      const unixTimestamp = Math.round(new Date().getTime() / 1000)
+      const message = `signature:${unixTimestamp}`
+      const prefix = `\x19Ethereum Signed Message:\n${message.length}`
+      const prefixedMessage = prefix + message
```

Comment:
nit: can use a template string to follow pattern in file

Reply:
Oh yeah good point. This was copied over from the request signature middleware.